app/src/main/java/edu/mit/mitmobile2/SoloMapActivity.java

```diff
@@ -84,6 +84,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         if (savedInstanceState != null) {
             mapViewExpanded = savedInstanceState.getBoolean(MAP_EXPANDED_KEY, false);
+            mapView.setMapViewExpanded(!mapViewExpanded);
```

Comment:
Why is the negated value for mapViewExpanded passed in? It should just be the original value